README.md

```diff
@@ -20,3 +20,5 @@
 - A diary sort of design that you can customise.
 
 - Some sort of pet management game which survives by consuming information that the player enters about their weekly development? Thinking a bit like Tamagotchi where if they leave it for too long, it will die.
+
+- Add the cpd to the attendance, ensuring it is completed weekly
```

Comment:
Could you describe what you mean by attendance in this context? Is this self-monitoring of attendance to scheduled learning activities, time in the studio, or some combination?

Reply:
to the weekly attendance on the learning space, requiring people to update it.

Reply:
Thanks for the clarification - can you please commit the change and I'll merge it in. 
:-)